[KED-1249] Fix unnecessary Kedro log message when Airflow command is used (#36)
quantumblacklabs/kedro-airflow: RELEASE.md

# Release 0.2.3

## Major features and improvements

> * Fix a bug of unnecessary Kedro logging message from runner when Airflow command is used.

Reviewer: ```suggestion
* Fix a bug that was logging unnecessary Kedro messages when running Airflow commands.
```
Maybe?
